pkgs/development/python-modules/tree-sitter-c-sharp/default.nix

```diff
@@ -0,0 +1,57 @@
+{
+  lib,
+  fetchFromGitHub,
+  buildPythonPackage,
+  pythonOlder,
+  cargo,
+  rustPlatform,
+  rustc,
+  setuptools,
+  wheel,
+  tree-sitter,
+}:
+
+let
+  version = "0.23.1";
+
+  src = fetchFromGitHub {
+    owner = "tree-sitter";
+    repo = "tree-sitter-c-sharp";
+    rev = "v${version}";
+    hash = "sha256-weH0nyLpvVK/OpgvOjTuJdH2Hm4a1wVshHmhUdFq3XA=";
+  };
```

Comment:
In your new files you pull version and src into the let blocks, in the above you pulled the version into a new let block but left src in. I'm trying to figure out the rhythm you're following with these.

Reply:
I tried to move the required variable into the `let ... in` block, so this was definitely a mistake on my part during the migration from `fetchCargoTarball`.

However, after we switched to `fetchCargoVendor`, the variable ended up being used again 😆